cfgov/paying_for_college/disclosures/scripts/update_colleges.py

```diff
@@ -1,4 +1,4 @@
-"""Update college data using the Dept. of Education's collegechoice api"""
+"""Update college data using the Dept. of Education's API."""
```

Comment:
Minor request: add [link to API documentation](https://collegescorecard.ed.gov/data/documentation/)?

Reply:
√